src/database/models/association.ts

```diff
@@ -45,3 +46,14 @@ UserCredential.belongsTo(User, {
   foreignKey: 'userId',
   as: 'user',
 });
+
+// UserMeta model
+User.hasMany(UserMeta, {
```

Comment:
User have only one entry of UserMeta in the table.

I`ve changed the relation from hasMany to hasOne